src/core/js/a11y/browserFocus.js

```diff
@@ -62,7 +62,9 @@ export default class BrowserFocus extends Backbone.Controller {
    */
   _onClick(event) {
     const $element = $(event.target);
-    if ($element.is('[aria-disabled=true]')) {
+    // search elemenent and parents for aria-disabled - see https://github.com/adaptlearning/adapt_framework/issues/3097
+    const isAriaDisabled = $element.closest('[aria-disabled=true]').length > 0;
```

Comment:
Just an FYI re: `> 0`, in practice `.length` will either be `0` or `1` since `$.closest` only ever returns one element for each source element.